cmd/podman/logs.go

```diff
@@ -2,20 +2,24 @@ package main
 
 import (
 	"fmt"
+	"io"
+	"os"
 	"strings"
 	"time"
 
-	"github.com/hpcloud/tail"
+	//"github.com/hpcloud/tail"
```

Comment:
If this is gone, can you remove it?

Reply:
Yeah, thats the vndr stuff I keep forgetting to do